Shop.BLL/Services/GameService.cs

```diff
@@ -0,0 +1,130 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Shop.BLL.Common.DataTransferObjects.Games;
+using Shop.BLL.Exceptions.BadRequestExceptions;
+using Shop.BLL.Exceptions.NotFoundExceptions;
+using Shop.BLL.Interfaces;
+using Shop.DAL.Models;
+using Shop.DAL.Interfaces;
+using System.Threading.Tasks;
+
+
+namespace Shop.BLL.Services
+{
+    public class GameService : IGameService
+    {
+        private readonly ILogger<GameService> _logger;
+        private readonly IMapper _mapper;
+        private readonly IGameRepository _gameRepository;
+        private readonly IUnitOfWork _unitOfWork;
+
+        public GameService(ILogger<GameService> logger, IMapper mapper, IGameRepository gameRepository, IUnitOfWork unitOfWork)
+        {
+            _logger = logger;
+            _mapper = mapper;
+            _gameRepository = gameRepository;
+            _unitOfWork = unitOfWork;
+        }
+
+        public async Task DeleteGameAsync(Guid id) 
```

Comment:
it is necessary to throw a token into all methods of classes and methods of repositories

Reply:
Done, but not sure